HTTPServerProject.Tests/Usings.cs

```diff
@@ -1 +1,11 @@
-global using Xunit;
\ No newline at end of file
+global using Xunit;
+global using System.Threading;
+global using System;
+global using System.Net;
+global using System.Net.Sockets;
+global using System.IO;
+global using HTTPServerProject.ReadStream;
```

Comment:
These bottom ones probably shouldn't be global but added with the files that need them.